tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrer.xtend

```diff
@@ -94,6 +106,28 @@ class TclJvmModelInferrer extends AbstractModelInferrer {
 			]
 		]
 	}
+	
+	private def JvmOperation createSetupMethod(TestSetup setup, Iterable<EnvironmentVariableReference> envParams) {
+		return setup.toMethod('setup', typeRef(Void.TYPE)) [
+			exceptions += typeRef(Exception)
+			annotations += annotationRef('org.junit.Before')
+			body = [
+				val output = trace(setup, true)
+				setup.contexts.forEach[generateContext(output.trace(it), #[], envParams)]
+			]
+		]
+	}
+
+	private def JvmOperation createCleanupMethod(TestCleanup cleanup, Iterable<EnvironmentVariableReference> envParams) {
+		return cleanup.toMethod('cleanup', typeRef(Void.TYPE)) [
+			exceptions += typeRef(Exception)
+			annotations += annotationRef('org.junit.After')
+			body = [
+				val output = trace(cleanup, true)
+				cleanup.contexts.forEach[generateContext(output.trace(it), #[], envParams)]
```

Comment:
How about surrounding each call into the fixture with a try/catch. Alternatively each fixture implementation to be used in a cleanup is be expected to be (very) robust